features/rendering.feature

```diff
@@ -19,7 +19,16 @@ Feature: Rendering
     And   I have a simple layout that contains "{{ content }}"
     When  I run jekyll build
     Then  I should get a non-zero exit-status
-    And   I should see "Liquid Exception.*Unknown tag 'INVALID' in.*_includes/invalid\.html" in the build output
+    And   I should see "Liquid Exception.+_includes/invalid\.html.+included in index\.html" in the build output
```

Comment:
I'd like to be able to test that `Unknown tag 'INVALID'` is still in the output – would that be possible? It can be another line (`And I should see...`) if the regexp is getting to weird.